examples/cifar10_cnn.py

```diff
@@ -99,3 +108,45 @@
                         steps_per_epoch=x_train.shape[0] // batch_size,
                         epochs=epochs,
                         validation_data=(x_test, y_test))
+
+# Save model and weights
+if not os.path.isdir(save_dir):
+    os.makedirs(save_dir)
+model_path = os.path.join(save_dir, model_name)
+weights_path = os.path.join(save_dir, weights_name)
+model.save(model_path)
+model.save_weights(weights_path)
```

Comment:
Didn't notice this earlier, but why save both the entire model and its weights, separately? The saved models already contains the weights. I suggest removing `save_weights`.